### DISCUSSION TITLE
Active Window Control

### DISCUSSION THREAD

**Discussion by joe23rep** [paulmcauley/klassy#66] [Q&A]
Is there a possibility to use Klassy with the active window control plasmoid?

i have it set up that active window control replaces my window buttons in the top bar when ever i maximize my windows. youre able to point active window control to a specific aurorae theme in its settings. it seems to work with svg files

**Comment by paulmcauley** [comment]
Please raise this with the developer of Active Window Control. If they wish to show the buttons properly, they should use the same technique as applet-window-buttons which seems to do the same thing.
